docs/labwc-config.5.scd

```diff
@@ -494,8 +494,8 @@ extending outward from the snapped edge.
 
 *<keyboard><keybind key="" onRelease="yes|no">*
 	*onRelease* makes keybind action take place on the release event of the
-	key. The most likely use-case for this the binding of a Super key to a
-	menu. Default: no. See example below:
+	key. The most likely use-case for this is the binding of a Super key to a
+	a menu. Default: no. See example below:
```

Comment:
extra 'a '

Reply:
Thanks. Changed it in-line with ahesfords suggestions - so "overtaken by events".